mne/viz/epochs.py

```diff
@@ -812,6 +813,11 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20, n_channels=20,
 
         .. versionadded:: 0.18.0
     %(show_scrollbars)s
+    bad_epochs_idx : array-like | None
+        Indices of bad epochs to show. No bad epochs to visualize if None.
+    log_labels : dataframe, shape (n_channels, n_epochs) | None
+        The bad segments to show in red and the interpolated segments
+        to show in green.
```

Comment:
This does not look like the API we discussed previously, it should just involve `epoch_colors` IIRC

https://github.com/mne-tools/mne-python/issues/6737#issuecomment-528440697

Reply:
yes, I am just trying to get it to work first :)